### PR TITLE
fix: updated validation when email permission is no

### PR DESCRIPTION
**Pull request by s-barrah**
fix: updated validation when email permission is no

### CODE REVIEW

File: tests/unit/Model/SQS/MarketingPreferences.model.js [hunk 1]
```diff
@@ -279,6 +279,47 @@ describe('Model/MarketingPreferencesModel', () => {
...
+      permissionEmail: 0,
+      permissionPost: null,
+      permissionPhone: null,
+      permissionSMS: null,
+      timestamp: '1562165588',
+    };
+
+    const model = new MarketingPreferencesModel(mockedData);
+
+    it('should validate the model', (done) => {
```
Comment (lebaz20) [inline tests/unit/Model/SQS/MarketingPreferences.model.js]: this was for testing, right ?

File: src/Model/SQS/MarketingPreference.model.js [hunk 2]
```diff
@@ -464,7 +464,8 @@ export default class MarketingPreference extends Model {
       const requestConstraintsClone = Object.assign({}, requestConstraints);
       if ((this.getPermissionEmail() !== null
         && this.getPermissionEmail() !== ''
-        && this.getPermissionEmail() !== '0')
+        && this.getPermissionEmail() !== '0'
+        && this.getPermissionEmail() !== 0)
```
Comment (lebaz20) [inline src/Model/SQS/MarketingPreference.model.js]:
```suggestion
        && this.getPermissionEmail() != 0)
```

Comment (s-barrah) [reply src/Model/SQS/MarketingPreference.model.js]: I know but you get an ESLINT warning

File: tests/unit/Model/SQS/MarketingPreferences.model.js (hunk 1 above)
Comment (s-barrah) [reply tests/unit/Model/SQS/MarketingPreferences.model.js]: yes

**Pull request merged**